### ISSUE TITLE
HumanTime might give next in sequency as past date

### ISSUE THREAD

**Issue by destotelhorus** [AgalmicVentures/HumanTime#1]
Example:

`import HumanTime
import datetime
basetime = datetime.datetime(2021, 8, 4, 9, 5, 0, 0) #5 minutes after 9am today, Wed
nexttime = HumanTime.parseTime("Wednesday at 9am", t=basetime)
print(basetime)
print(nexttime)`

will output:
`2021-08-04 09:05:00
2121-08-04 09:00:00`

**Comment by AgalmicVentures** [comment]
Good catch! And thank you for submitting this. I agree that it should return Wednesday 2021-08-11.

The inconsistency is due to the time of day being applied too late during parsing. Unfortunately, fixing this will be a challenge given the way the library is structured. Let me think this over, and I will try to get a fix out soon,

Ian
